src/renderer/state/upload/logics.ts

```diff
@@ -220,7 +220,10 @@ const initiateUploadLogic = createLogic({
       return;
     }
 
-    dispatch(initiateUploadSucceeded(action.payload));
+    dispatch(batchActions([
+      initiateUploadSucceeded(action.payload),
+      resetUpload()
```

Comment:
Any straight forward way to unit test this?

Reply:
Good idea, added